cppfd/CMakeLists.txt

```diff
@@ -7,7 +7,9 @@ project(cfd-mini-app)
 find_package(Kokkos REQUIRED)
 find_package(KokkosKernels REQUIRED)
 
-find_package(VTK REQUIRED COMPONENTS vtkCommonCore vtkIOXML)
+if(OUTPUT_VTK_FILES)
```

Comment:
This option `OUTPUT_VTK_FILES` should be visible in the CMake GUI when running `ccmake`